certif/tests/acceptance/session-finalization_test.js

```diff
@@ -261,6 +261,79 @@ module('Acceptance | Session Finalization', function (hooks) {
               .exists();
           });
         });
+
+        module('when there is uncompleted report', function () {
+          module('when we add a certification issue report on completed report', function () {
```

Comment:
`when there is uncompleted report` et `when we add a certification issue report on completed report`
Je trouve pas ça super clair. 🤔

Reply:
Pareil, je changerais aussi "when there is an uncompleted report" ou "when there is already an uncompleted report"

Reply:
En gros si tu remontes un peu l'arbre ca fait `when there is completed report` > `when there is uncompleted report` > `when we add a certification issue report on completed report` et `when we add a certification issue report on uncompleted report`
C'était histoire de reprendre les module existant mais sinon tant pis j'en créé un à part qui dit `when there are completed report and uncompleted report` ?

Reply:
Ouais ça serait sans doute plus lisible donc mieux comme contexte